cmd/archive_upload_client/client.go

```diff
@@ -38,31 +63,13 @@ func newConn(host string) (*grpc.ClientConn, error) {
 	cred := credentials.NewTLS(&tls.Config{
 		RootCAs: systemRoots,
 	})
-	opts = append(opts, grpc.WithTransportCredentials(cred))
+	opts = append(opts, grpc.WithTransportCredentials(cred), grpc.WithPerRPCCredentials(
```

Comment:
it looks like the maxSize change isn't included in your git branch base?
Possibly easiest to just add the const at the top and set the option here.

Reply:
Oops - probably lost in the merge. Fixed.